examples/svelte-counter/src/services/counter.ts

```diff
@@ -10,22 +10,22 @@ export const counterApi = createApi({
     entityTypes: ['Counter'],
     endpoints: (build) => ({
         getCount: build.query<CountResponse, void>({
-            query: () => `count`,
+            query: () => `/count`,
             provides: [{ type: 'Counter' }],
         }),
         incrementCount: build.mutation<CountResponse, number>({
-            query: (amount) => ({
-                url: `increment`,
-                method: 'PUT',
-                body: JSON.stringify({ amount }),
+            query: (body) => ({
+                url: `/increment`,
+                method: 'PUT', 
+                body, // Very clever!
```

Comment:
But won't increment and decrement behave differently now? One POSTING `5` and the other `{amount:5}`

Reply:
Yea, good catch. We committed while discussing & testing random things 🎉